packages/monaco/package.json

```diff
@@ -7,7 +7,7 @@
     "@theia/editor": "1.13.0",
     "@theia/filesystem": "1.13.0",
     "@theia/markers": "1.13.0",
-    "@theia/monaco-editor-core": "^0.20.0",
+    "@theia/monaco-editor-core": "^0.23.0-next-82e8ea39fc101d63.2",
```

Comment:
why there is only next version for this?

Reply:
As long as this PR is not approved then I'm not publishing release version `0.23.0` to npm - in case there will be some changes needed.
Also, `@theia/monaco-editor-core` needs also to be approved.